lib/src/constants/constants.dart

```diff
@@ -0,0 +1,30 @@
+class SystemProgram {
```

Comment:
There shouldn't be instantiable classes that are not supposed to be instatiated.

It should be either top-level constants, like `systemProgramId` or at least abstract classes with private constructors:

```dart
abstract class SystemProgram {
  SystemProgram._();

  static const id = '11111111111111111111111111111111';
}
```

Reply:
Understood! Can you please just approve this and merge it? I will do it in my working tree as I had to merge this already with it so doing modifications to it can make things messy!